### PR TITLE
install/kubernetes: Fix experimental-install.yaml for Hubble

### PR DESCRIPTION
**Pull request by gandro**
This change fixes the `experimental-install.yaml`, which was broken due
to the certgen changes introduced with #13666. The version v0.1.2 of
certgen accidentally changed the behavior of the
`hubble-generate-certs`, where it would try (and fail) to load the
unrelated clustermesh-apiserver CA, causing the job to end up in a
CrashLoopBackoff.

This PR bumps certgen to v0.1.3, which contains a fix for the above
issue and brings in some additional smaller improvements for Hubble,
such as:

  - The Hubble CA can now be loaded from a secret instead of having to
    regenerate it each time. This improves connectivity for Hubble
    during certificate renewal, as now the old and new certificates
    share the same CA.
  - The `--cilium-namespace` flag can now be used to control the
    namespace for the Hubble certificate secrets and configmaps.

Signed-off-by: Sebastian Wicki <sebastian@isovalent.com>

### CODE REVIEW

File: install/kubernetes/cilium/templates/_clustermesh-apiserver-generate-certs-job-spec.tpl [hunk 1]
```diff
@@ -15,6 +15,8 @@ spec:
           command:
             - "/usr/bin/cilium-certgen"
           args:
+            - "--cilium-namespace={{ .Release.Namespace }}"
```
Comment (jrajahalme) [inline install/kubernetes/cilium/templates/_clustermesh-apiserver-generate-certs-job-spec.tpl:18]: Thanks for fixing this as well!

File: install/kubernetes/cilium/templates/_hubble-generate-certs-job-spec.tpl [hunk 2]
```diff
@@ -17,61 +17,47 @@ spec:
           {{/* Because this is executed as a job, we pass the values as command line args instead of via config map,
                 this allows users to inspect the values used in past runs by inspecting the completed pod */ -}}
           args:
+            - "--cilium-namespace={{ .Release.Namespace }}"
```
Comment (jrajahalme) [inline install/kubernetes/cilium/templates/_hubble-generate-certs-job-spec.tpl:20]: I did not notice that certgen can use this for hubble already, most likely just missed it.

**Review by jrajahalme — APPROVED** [review]

File: install/kubernetes/cilium/templates/_hubble-generate-certs-job-spec.tpl (hunk 2 above)
Comment (gandro) [reply install/kubernetes/cilium/templates/_hubble-generate-certs-job-spec.tpl:20]: Ah yes, this is very recent :) https://github.com/cilium/certgen/pull/10

**Comment by gandro** [comment]
test-me-please

**Review by michi-covalent — APPROVED** [review]

**Review by sayboras — APPROVED** [review]
LGTM :100: 

I observed that there are a few similar issues recently, maybe we can bring `experimental-install` github action back.

Related slack discussion https://cilium.slack.com/archives/C2B917YHE/p1598950373002000, still need to think of a way to decouple code changes + helm changes.

**Pull request merged**